src/docs/03-concepts/08-cross-dc-replication.md

```diff
@@ -87,6 +87,79 @@ The following command can be used to failover Global :domain:Domain: *my-domain-
 $ cadence --do my-domain-global d up --ac dc2
 ```
 
+## Running Locally
+
+The best way is to use Cadence [docker-compose](https://github.com/uber/cadence/tree/master/docker):
+`docker-compose -f docker-compose-multiclusters.yml up`
+
+
+## Running in Production
+
+Enable global domain feature needs to be enabled in [static config](/docs/operation-guide/setup/#static-configs).  
+
+Here we use clusterDCA and clusterDCB as an example. We pick clusterDCA as the master cluster.
```

Comment:
I will update this name soon. Could we use primary cluster?

Reply:
yeah sure, will update it. 
btw when you update the configuration, we should have some release notes to tell people about it. and it's better to have a compatible period(minor) version with warnings logs before breaking.